server/public/styles.css

```diff
@@ -318,6 +319,72 @@ h4 {
   margin-left: 15px;
 }
 
+.app-drawer {
+  background-color: #ffd700;
+  width: 50%;
```

Comment:
Please set this to something like 300px instead of a percentage. It looks way too huge on desktop.